Swift Code Challenge/Views/MapView.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MapView.swift
+//  Swift Code Challenge
+//
+//  Created by Luiz Eduardo Mello dos Reis on 30/06/22.
+//
+
+import SwiftUI
+import MapKit
+
+struct MapView: UIViewRepresentable {
```

Comment:
SwiftUI has a `Map` object which would be better to use and customize than creating a representable unless we need some specific functionality that the native objects won't give us.